packages/venia-concept/src/components/ProductFullDetail/ProductFullDetail.js

```diff
@@ -18,7 +18,7 @@ import isProductConfigurable from 'src/util/isProductConfigurable';
 
 const Options = React.lazy(() => import('../ProductOptions'));
 
-class ProductFullDetail extends Component {
+export class ProductFullDetail extends Component {
```

Comment:
@jimbo - @sirugh and I wanted your opinion on this change. A common pattern in Venia is to use container components that handles the redux and classify HoCs, but this component currently does it inline, and that change would not be backward compatible. This change allows us to more easily unit test, and we know there are alternatives, but this seemed like the most simple backward compatible solution. Would you prefer either of the alternatives below?

1) Add container component, BIC approved.
2) Mock `classify`, `connect`, and `compose` and leave implementation as-is (ran into issues with this, so would need some guidance).

Reply:
3. export the unwrapped class but as a name that identifies it as such (ie.` UnwrappedProductFullDetail`)

Reply:
I don't think we should be afraid of breaking changes at this stage of the game (the next release will already be a major if #1169 goes in). I'd prefer the consistency of a container component here.

Reply:
Neither of us really liked it, very confusing that `import ProductFullDetail` would be different from `import { ProductFullDetail }`. Made this change. Should we add this pattern to the standards and conventions wiki article?